Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -521,6 +562,38 @@ void addReactantNeighborsToProduct(
             const Bond *origB =
                 reactant.getBondBetweenAtoms(lreactIdx, *nbrIdx);
             addMissingProductBonds(*origB, product, mapping);
+          } else {
+            // both mapped atoms are in the match.
```

Comment:
This logic is mind bending and should probably be in the docs.

Am I right below?

```
([C:1].[C:2])>>([C:1].[C:2])  # do nothing
([C:1][C:2])>>([C:1].[C:2]) # break bond
```

However we aren't treating "." special here, so the rule is effectively the same as:

```
([C:1][C:3][C:2])>>([C:1].[C:2])  # explicit bond break?
```